packages/project-config/src/test/builds_test.ts

```diff
@@ -28,20 +27,25 @@ suite('builds', () => {
       assert.equal(isValidPreset('es6'), false);
       assert.equal(isValidPreset('js-compile'), false);
       assert.equal(isValidPreset(''), false);
-      assert.equal(isValidPreset(null), false);
-      assert.equal(isValidPreset(undefined), false);
-      assert.equal(isValidPreset(0), false);
-      assert.equal(isValidPreset(1), false);
+      assert.equal(isValidPreset(null as any), false);
```

Comment:
this should fail to lint. need to add

`// tslint:disable-next-line: no-any Explicitly testing invalid code.`